js/scripts.js

```diff
@@ -0,0 +1,202 @@
+const builder = {};
+
+// doc ready function
+function ready(callback){
+    // in case the document is already rendered
+    if (document.readyState!='loading') callback();
+    // modern browsers
+    else if (document.addEventListener) document.addEventListener('DOMContentLoaded', callback);
+    // IE <= 8
+    else document.attachEvent('onreadystatechange', function(){
+        if (document.readyState=='complete') callback();
+    });
+}
+
+// on doc ready call init
+ready(function(){
+    builder.init();
+});
+
+builder.init = () => {
```

Comment:
I like how you've separated these functionalities into their own methods to make it more modular.